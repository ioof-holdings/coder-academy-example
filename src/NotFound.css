.NotFound h1 {
  color: white
}