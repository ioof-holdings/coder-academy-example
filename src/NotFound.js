import React from "react";
import "./NotFound.css";

const NotFound = () => (
  <div data-testid="not-found" className="NotFound">
    <h1>Whoops!</h1>
  </div>
);

export default NotFound;
