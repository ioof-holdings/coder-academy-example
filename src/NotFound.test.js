import React from "react";
import { render, cleanup } from "@testing-library/react";
import NotFound from "./NotFound";

afterEach(cleanup);

test("should render not found component", () => {
  const { container } = render(<NotFound />);
  expect(container).toMatchSnapshot();
});
