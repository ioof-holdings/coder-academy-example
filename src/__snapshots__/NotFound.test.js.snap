// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render not found component 1`] = `
<div>
  <div
    class="NotFound"
    data-testid="not-found"
  >
    <h1>
      Whoops!
    </h1>
  </div>
</div>
`;
